### DISCUSSION TITLE
[Fundamentos do C# - System] System é um tipo ou uma biblioteca?

### DISCUSSION THREAD

**Discussion by Schinwinkwinsky** [balta-io/forum#536] [Tira Dúvidas] [answered]
Bom dia, jovens! Na aula System de Fundamentos do C#, Balta comenta que todos os tipos derivam de System como se este fosse um tipo. System é um tipo ou uma biblioteca?

**Comment by henriqueweege** [comment, marked as answer]
Oi, @Schinwinkwinsky ! Tudo bem?
Então, segundo [essa doc](https://learn.microsoft.com/pt-br/dotnet/api/system?view=net-8.0) o System é um namespace que contém classes base, como os "built-in types" (int, string, etc), por exemplo. 
Valeeeu!
